orchagent/portsorch.cpp

```diff
@@ -1796,6 +1796,47 @@ bool PortsOrch::setHostIntfsOperStatus(const Port& port, bool isUp) const
     return true;
 }
 
+bool PortsOrch::createVlanHostIntf(Port& vl, string hostif_name)
+{
+    SWSS_LOG_ENTER();
+
```

Comment:
Add check if VLAN already has host_intf and fail if so

Reply:
as discussed